api/src/main/java/org/ihtsdo/rvf/controller/AssertionGroupController.java

```diff
@@ -41,10 +42,10 @@
 
 	@Autowired
 	private AssertionService assertionService;
-	
+
```

Comment:
Fixing their whitespace is probably outside our remit

Reply:
That's probably just an artefact of an editor configuration